test/tests/cache.test.js

```diff
@@ -77,12 +77,12 @@ describe('Handling cache purges', () => {
       expect(body).to.contain('Custom endpoint')
       expect(res.statusCode).to.equal(200)
 
-      request.get(`${uri}/purge/${route}`, (err, res, body) => {
+      request.get(`${uri}/purge/${route}`, async (err, res, body) => {
         const cacheApi = cacheManager.getCache('api')
         expect(body).to.contain(JSON.stringify(purgeResp))
         expect(res.statusCode).to.equal(200)
-        expect(cacheApi.keys()).to.not.contain('pages?slug=test')
```

Comment:
I could've left this as `await cacheApi.keys()`, but - and I don't know whether it's because I've been coding too much this week` it took my ages to work out what going on here. Maybe it depends on your mental model. I could switch these back.